sample.env

```diff
@@ -1,8 +1,5 @@
-LOG_LEVEL=0
-BRAZE_MAX_REQ_COUNT=5
-################################################
-# Marketo specific env variables
-################################################
-# in seconds
-MARKETO_LEAD_CACHE_TTL=10
-MARKETO_AUTH_CACHE_TTL=10
\ No newline at end of file
+PORT=9090 # application's port
+ENABLE_FUNCTIONS=true # user transformation functions
+CLUSTER_ENABLED=true # cluster mode
+NUM_PROCS=1 # Number of workers in the cluster mode
```

Comment:
Why are running transformer in cluster mode in such case?

Reply:
It's similar to the production environment configuration.

Reply:
But we need to validate it for sure.